python/lsst/daf/butler/cli/butler.py

```diff
@@ -253,7 +253,7 @@ def get_command(self, context, name):
         return doImport(commands[name][0] + "." + cmdNameToFuncName(name))
 
 
-@click.command(cls=LoaderCLI)
+@click.command(cls=LoaderCLI, context_settings=dict(help_option_names=['-h', '--help']))
```

Comment:
General comment but in daf_butler we always prefer double quotes to single quotes.

Reply:
yeah. This was copy/paste and I didn't notice the quotes, will fix.